lexicons/com/atproto/admin/getSubjectState.json

```diff
@@ -0,0 +1,39 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.admin.getSubjectState",
```

Comment:
I see what you mean about naming being hard here—"subject state" is a little loaded.  Some other ideas to replace: "state": flags, statuses, notices, advisories.